WcaOnRails/config/locales/ru.yml

```diff
@@ -1402,11 +1403,11 @@ ru:
     nearby_competitions:
       #original_hash: 241801a
       competitions:
-        other: '%{count} WTF'
-        many: '%{count} соревнований'
-        few: '%{count} соревнования'
-        one: '%{count} соревнование'
         zero: Нет соревнований
+        one: '%{count} соревнование'
+        few: '%{count} соревнования'
+        many: '%{count} соревнований'
+        other: '%{count} WTF'
```

Comment:
That shouldn't be visible, because :other key is not used. We check it with Philippe:

>     (byebug) (0..100000).each { |c| if (I18n.t('toto.akey', count: c) == "other") then puts c end }
>      0..100000

[Here](https://github.com/svenfuchs/rails-i18n/blob/master/lib/rails_i18n/common_pluralizations/east_slavic.rb) is the code, one can check that :other branch is never reached.